c2corg_ui/static/partials/imageuploader.html

```diff
@@ -0,0 +1,117 @@
+  <div ng-if="uplCtrl.files.length === 0"
+       ngf-drop
+       ngf-select
+       ng-model="uplCtrl.files"
+       class="drop-box" 
+       ngf-drag-over-class="'dragover'"
+       ngf-multiple="true"
+       ngf-keep="'distinct'" 
+       ngf-fix-orientation="true"
+       ngf-allow-dir="true"
+       accept="image/*" 
+       ngf-pattern="'image/*'" translate>
+    <span translate>Drop images here or click to upload</span>
+  </div>
+
+  <div ng-show="uplCtrl.files.length > 0" class="images-container" app-loading>
+    <div ng-repeat="file in uplCtrl.files track by $index" class="uploaded-image {{$index}}" ng-show="file">
+
+      <div class="img-container" style="background-image: url({{file.src}})">
+        <!-- EXIF INFO-->
+        <div class="exif" ng-style="file.metadata.FocalLength && {'opacity': 1} || {'opacity' : 0}">
+          <p ng-if="file.metadata.DateTime"><span class="glyphicon glyphicon-calendar"></span><span>{{file.metadata.DateTime}}</span></p>
+          <p><span class="glyphicon glyphicon-certificate"></span><span>{{file.metadata.FocalLength}}f</span>&nbsp;
+            <span ng-if="file.metadata.ShutterSpeedValue">1/{{file.metadata.ShutterSpeedValue}}s</span> &nbsp; 
+            <span><b>ISO:</b> <span>{{file.metadata.PhotographicSensitivity}}</span></span>
+          </p>
+          <p ng-if="file.metadata.geo"><span class="glyphicon glyphicon-map-marker"></span><b>{{file.metadata.geo.label}}</b></p>
+        </div>
+        <span class="glyphicon glyphicon-remove-circle remove-image-btn" type="button" ng-click="uplCtrl.deleteImage($index)" ng-show="file.progress >= 100"></span>
+      </div>
+
+      <div class="img-data">
+        <!-- TITLE-->
+        <input type="text" class="form-control" ng-model="file.metadata.title" placeholder="title" ng-change="uplCtrl.rename(file, file.metadata.title)">
+
+        <div class="image-attributes btn-group" ng-show="file.progress >= 100">
+          <!-- GEOLOCATION-->
+          <div class="btn-group">
+            <button class="btn btn-default dropdown-toggle" ng-class="{'btn-success': file.metadata.geo}" type="button" data-toggle="dropdown">
+              <span class="glyphicon glyphicon-globe"></span>
+            </button>
+            <div class="dropdown-menu geo">
+              <p translate ng-if="!file.metadata.geo">You can add geolocation info by typing in the input field.</p>
```

Comment:
It does add the geoinfo to file.metadata.geo. If we want to go further, we can add a map tool to georeference the images. 
Btw, I haven't tried anything with images that are georeferenced. 

> By the way is it really relevant that this tool searches on routes and outings?
This 'issue' is fixed in #328, I didn't want to fix it twice, it should work when both are merged.